scripts/wipe_sensitive_info_db.py

```diff
@@ -68,7 +35,7 @@ def main(argv):
     ):
       sys.exit(0)
 
-  wipe_db(argv[1])
+  wipe_db_path(argv[1], FLAGS.keep_beds)
```

Comment:
i would rather use FLAGS for both arguments, i think it s more readable and more consistent

Reply:
Hmmm OK but the filename is non-optional, whereas `keep_beds` is optional and therefore a flag?

Reply:
You can make the flag required with absl.  Agreed the flags should generally be kept optional according to argparse/general unix culture, but I also tend to like explicit flags vs. positional arguments.

Reply:
@dulacarnold @olivierteboul OK, filename is now a required flag